test/fluree/db/shacl/shacl_basic_test.clj

```diff
@@ -1425,6 +1425,90 @@ WORLD!")
           (is (= "Subject :ex/john path [:ex/birthYear] violates constraint :sh/pattern of shape _:fdb-3 - value \":ex/ref\" does not match pattern \"(19|20)[0-9][0-9]\"."
                  (ex-message db-ref-value))))))))
 
+(deftest language-constraints
```

Comment:
These test needs more commentary. What are you testing? What do you expect to happen in these tests?